lib/Habrok.js

```diff
@@ -78,13 +78,17 @@ Habrok.prototype.request = function request(raw, opts) {
     } else if (errorCode === ECONNRESET || statusCode > 299) {
       if ((!err && self.RETRY_CODES.indexOf(statusCode) === -1) || attempt >= self.RETRIES) {
         debug('fail', req);
-
+        if (has(options, 'debugRequest')) options.debugRequest({ attempt });
```

Comment:
Use brackets with if statements. Surprised that didn't show up in ESLint

```js
if (has(options, 'debugRequest')) {
  options.debugRequest({ attempt })
}

Reply:
But otherwise looks good. I think the arguments are appropriate. Let's talk about when it is appropriate to use an object for the args and when you should send individual args.